main.py

```diff
@@ -206,17 +297,24 @@ def main():
         help="source queue to process",
     )
     parser.add_argument(
-        "--dst-queue-beehive",
+        "--dst-exchange-beehive",
         default=getenv("DST_QUEUE_BEEHIVE", "to-beehive"),
```

Comment:
u wanna change this to `DST_QUEUE_NODE`

its a bit odd to have

`DST_QUEUE_BEEHIVE`
and 
`DST_EXCHANGE_NODE`

Reply:
Good catch - that should have been `DST_EXCHANGE_BEEHIVE`.

Reply:
(The technical reason for the change in language is in that RabbitMQ, we don't publish directly to queues but to exchanges with queues bound to them.)